### DISCUSSION TITLE
QR Code way to complex

### DISCUSSION THREAD

**Discussion by franz101** [Digitaler-Impfnachweis/certification-apis#48] [General]
I can see many problems arise due to people having a small phone or a dusty / dirty screen.
Imagine scanning at a festival under extreme light conditions. 

What alternative ways are there to authenticate?

**Comment by gr1zz0** [comment]
Maybe printed (and laminated) paper?

**Comment by TimLinsenmayer** [comment]
It might be possible to go with a similar authentification to TeamViewer, with a (maybe one-time) code and a tan (generated on end device upon request for X minutes).
Would require enabled internet though. Otherwise in the long term the app could integrate into Google Pay / Wallet #9 and authenticate using NFC terminals.

@gr1zz0 is probably the best use case currently if you want it fast and reliable (even without signal)
